sections/futures/FundingChart.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect } from 'react';
+import { LineChart, XAxis, Line, ResponsiveContainer, YAxis, Tooltip } from 'recharts';
+import styled from 'styled-components';
+import { useTheme } from 'styled-components';
+
+import { fetchFundingRates } from 'state/futures/actions';
+import { selectMarketInfo } from 'state/futures/selectors';
+import { useAppDispatch, useAppSelector } from 'state/hooks';
+import { formatChartTime } from 'utils/formatters/date';
+
+import FundingChartTooltip, { formatFundingRate } from './FundingChartTooltip';
+
+const FundingChart = () => {
+	const theme = useTheme();
+	const dispatch = useAppDispatch();
+	const marketInfo = useAppSelector(selectMarketInfo);
+	const marketFundingRates = useAppSelector(({ futures }) => futures.marketFundingRates);
+
+	useEffect(() => {
+		if (marketInfo?.asset) {
+			dispatch(fetchFundingRates(marketInfo.asset));
```

Comment:
Might be good to use `usePollAction` here instead with an hourly interval

Reply:
Added in the latest commit